mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -284,7 +284,10 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         $('.grade').data(response);
     }
 
-    function calculate_results(handler_name, callback) {
+    function calculate_results(handler_name, successs_callback, error_callback) {
+        if (typeof error_callback === 'undefined'){
+            error_callback = function () {};
```

Comment:
I think you can just pass undefined as the error callback, jquery will do
the right thing